package.json

```diff
@@ -31,6 +31,9 @@
     "request": "^2.67.0",
     "tape": "^4.2.2"
   },
+  "engines": {
+    "node": ">=4.2.1"
```

Comment:
@bsudekum v4.2.0 introduced a bug that was fixed in https://github.com/nodejs/node/commit/b3cbd133400a73a54ca8eab5165efe866b26b4c5, and IIRC there was another bug in early v4.0.x releases. I could track down exactly which versions are incompatible, but this was simpler.